src/sync.d

```diff
@@ -445,9 +445,36 @@ final class SyncEngine
 			// Check if there is an interrupted upload session
 			if (session.restore()) {
 				log.log("Continuing the upload session ...");
+				string sessionFilePath = session.getSessionFilePath();
```

Comment:
About this variable name: see comment on upload.d https://github.com/abraunegg/onedrive/pull/1399/files#r611176161

Reply:
good point .. yes .. function name should not be named the same

Reply:
Resolved by https://github.com/abraunegg/onedrive/pull/1399/commits/9cdf81b074715c9f07a5be92d99d7bf5c22cc324